WooCommerce/Resources/id.lproj/Localizable.strings

```diff
@@ -1,4 +1,4 @@
-/* Translation-Revision-Date: 2022-04-26 09:54:04+0000 */
+/* Translation-Revision-Date: 2022-04-26 16:22:52+0000 */
```

Comment:
🤔 I wonder why the date changed but we have no change in the `.strings`.

And just for reference, 22/4/26 is _not_ the day in which the file was downloaded.